lib/taste_tester/client.rb

```diff
@@ -125,14 +125,62 @@ def upload
 
     private
 
+    def tar_content(list, name)
+      full_paths = list.map do |path|
+        "#{TasteTester::Config.repo}/#{path}"
+      end
+      out = full_paths.map do |path|
+        if path.start_with?('/')
+          path = path[1..-1]
+        end
+        "--transform 's,#{path},#{name},'"
```

Comment:
This whole method needs lots of comments... 

The method as a whole does - "tar_content" implies it returns the contents of a tar, which this does not, so what does this do? (yes I know it generates parts of the tar command, sorta, but a comment needs to explain that).

Then a comment at each block... there's a lot of dense stuff here...

Reply:
This code was removed